.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,34 @@
+name: HOPR Nightly Integration Test
+on: workflow_dispatch
+  #    schedule:
+  #   - cron: '0 12 * * *' 
+      # workflow_dispatch:
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - name: Setup Google Cloud Credentials
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '290.0.1'
+          service_account_key: ${{ secrets.GOOGLE_HOPRASSOCIATION_CREDENTIALS_REGISTRY }}
+          export_default_credentials: true
+      - name: Set Project for Google Cloud HOPR Association
+        run: gcloud config set project ${{ secrets.GOOGLE_HOPRASSOCIATION_PROJECT }}
+      - name: Run start script
+        run: |
+          bash scripts/nightly.sh
+        env:
+          FUNDING_PRIV_KEY: ${{ secrets.FUNDING_WALLET_PRIVATE_KEY }}
+          RPC: ${{ secrets.FUNDING_RPC_MATIC_URL }}
```

Comment:
Let's change the default to be `bnb`. Ideally we use a matrix strategy to deploy all our networks, for the time being just update the default one and create a new task in #875 to implement a multiple network run down. [Here's](https://github.com/hoprnet/hopr-devops/blob/master/.github/workflows/terraform.yml#L21-L32) and example on how to do this via dynamic secrets.

Reply:
Are we really planning on running on all these chains long term? Or can we settle on a single one for now.

Reply:
do we have a FUNDING_RPC_BNB_URL secret?